package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "TodoList",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+
+    "dev": "webpack --mode development",
+
+    "build": "webpack --mode production",
+
+    "start": "webpack-dev-server --mode development --open"
```

Comment:
### [Optional]

You added `webpack-dev-server` which is great but you didn't include it in `devDependencies`